ksql-engine/src/main/java/io/confluent/ksql/util/PersistentQueryMetadata.java

```diff
@@ -134,6 +140,11 @@ public PhysicalSchema getPhysicalSchema() {
       final QueryId queryId,
```

Comment:
this shouldn't take in queryId if it's not being used! I think the same is true of `contextStacker`